templates/vue/src/components/modals/NodeModal/index.vue

```diff
@@ -432,6 +435,11 @@ export default {
     isMultiContentNodeChild() {
       return this.parent && this.parent.mediaType == "multi-content"
     },
+    isPageMultiConentNodeChild() {
+      return (
+        !!this.isMultiContentNodeChild && this?.parent?.presentationStyle === "page"
```

Comment:
reduce to `this.isMultiContentNodeChild && ...`

Reply:
Turns out this is needed, otherwise it might return null sometimes